azure/code/train_rapids.py

```diff
@@ -98,7 +98,7 @@ def main():
         print( f'\n CV fold { i_train_fold } of { 5 }\n' )
 
         # split data
-        X_train, X_test, y_train, y_test, _ = azure_ml.split_data(X, y, random_state =77)
+        X_train, X_test, y_train, y_test, _ = azure_ml.split_data(X, y, random_state = i_train_fold)
```

Comment:
This breaks PEP 8, regarding spaces for un-annotated function parameters. Should be func(a=b, c=d)... etc.. to avoid style complaints.